lib/cdo.rb

```diff
@@ -110,6 +110,14 @@ def site_host(domain)
       host
     end
 
+    def dashboard_site_host
+      site_host('studio.code.org')
+    end
+
+    def pegasus_site_host
```

Comment:
> The names of the underlying functions are a little confusing, but this specific change seems reasonable.

agreed these names are confusing. the best name would be `dashboard_host`, since [host means "hostname and port"](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Host), but `dashboard_host` is already in use in our config: https://github.com/code-dot-org/code-dot-org/blob/a22d066f6e2eb87350cf473db862c509542dca8d/config.yml.erb#L388